tfhe/src/integer/gpu/mod.rs

```diff
@@ -2891,6 +2891,156 @@ pub unsafe fn unchecked_unsigned_scalar_div_rem_integer_radix_kb_assign_async<
     update_noise_degree(remainder, &cuda_ffi_remainder);
 }
 
+#[allow(clippy::too_many_arguments)]
+#[allow(clippy::fn_params_excessive_bools)]
+/// # Safety
+///
+/// - [CudaStreams::synchronize] __must__ be called after this function as soon as synchronization
+///   is required
+pub unsafe fn unchecked_signed_scalar_div_rem_integer_radix_kb_assign_async<
+    T: UnsignedInteger,
+    B: Numeric,
+>(
+    streams: &CudaStreams,
```

Comment:
Ok now there are really too many arguments to this function, we should pass the data about the scalar in struct as we discussed earlier. We could merge this PR and right after you do the refactor for all scalar_mul, scalar_div, signed_scalar_div, scalar_div_rem, signed_scalar_div_rem, how does that sound?